src/GraphQL/Execution/SubscriptionExecutionStrategy.cs

```diff
@@ -101,6 +101,9 @@ protected virtual async Task<IObservable<ExecutionResult>> ResolveEventStreamAsy
                     throw new InvalidOperationException($"Subscriber not set for field '{node.Field.Name}'.");
                 }
 
+                // The IServiceProvider instance will be disposed at this point, set it to null insted of exposing the disposed object
```

Comment:
```suggestion
                // The IServiceProvider instance will be disposed at this point, set it to null instead of exposing the disposed object
```